src/client/pydaos/raw/daos_api.py

```diff
@@ -2363,6 +2363,7 @@ def get_function(self, function):
             # that daos_init() has been called before
             # invoking anything.
             self._dc = DaosClient()
+            self._dc._atfork = 0
```

Comment:
> protected-access, Access to a protected member _atfork of a client class

Since this is pydaos raw we can just ignore like (with two spaces before the comment)
```suggestion
            self._dc._atfork = 0  # pylint: disable=protected-access
```